README.md

```diff
@@ -24,5 +24,19 @@ Run `./dss-api` in this directory.
 #### Running tests
 Run `make test` in this directory.
 
+#### Configuring cloud-specific access credentials
```

Comment:
thanks so much for writing this up.  Setting the bucket permissions up right was challenging, but probably not required for this.